### PR TITLE
Productivity Limitations, closes #104, closes #109

### LINKED ISSUE #104: Stone brick doesn't allow you to select productivity modules
Stone brick for some reason only shows speed modules in the furnaces

### LINKED ISSUE #109: Update icon colors
Factorio has tweaked the high res icons a couple times since released in 0.18.27:
https://wiki.factorio.com/Version_history/0.18.0#0.18.29
https://wiki.factorio.com/Version_history/0.18.0#0.18.30

### PR DESCRIPTION
**Pull request by dcbroad3**

### CODE REVIEW

File: src/app/models/data/item.ts [hunk 1]
```diff
@@ -17,9 +17,9 @@ export enum ItemId {
   CopperOre = 'copper-ore',
   CopperPlate = 'copper-plate',
   CrudeOil = 'crude-oil',
-  EfficiencyModule = 'efficiency-module',
-  EfficiencyModule2 = 'efficiency-module-2',
-  EfficiencyModule3 = 'efficiency-module-3',
+  EfficiencyModule = 'effectivity-module',
+  EfficiencyModule2 = 'effectivity-module-2',
```
Comment (hhrhhr) [inline src/app/models/data/item.ts:21]: If I understand correctly, then the use of "hardcoded" variables does not allow working with modules added by mods?

Comment (dcbroad3) [reply src/app/models/data/item.ts:21]: There are a lot of areas in the code that make assumptions that you are using the base mod. Rooting them out will take time, my first step is to rip out the oil and uranium utilities, issue #90, and replacing them with a generic solver that can handle complex recipes from mods.

Comment (bbarry) [reply src/app/models/data/item.ts:21]: And certainly: PRs welcome 😄

File: src/app/store/settings/settings.reducer.ts [hunk 2]
```diff
@@ -23,6 +30,11 @@ export interface SettingsState {
   theme?: Theme;
 }
 
+export const loadTheme = () => {
+  const lsTheme = localStorage.getItem(LocalStorageKey.Theme);
+  return lsTheme ? (lsTheme as Theme) : Theme.DarkMode;
+};
```
Comment (bbarry) [inline src/app/store/settings/settings.reducer.ts:36]: 👍

**Pull request merged**